test/built-ins/Temporal/Duration/prototype/with/argument-singular-properties.js

```diff
@@ -1,20 +1,15 @@
-// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
 esid: sec-temporal.duration.prototype.with
-description: >
-  The durationLike argument must contain at least one correctly spelled property
+description: Singular properties in the property bag are always ignored
 features: [Temporal]
 ---*/
 
-let d = new Temporal.Duration(1, 2, 3, 4, 5);
+const instance = new Temporal.Duration(1, 2, 3, 4, 5, 6, 7, 987, 654, 321);
 
 [
-  {},
-  [],
-  () => {},
```

Comment:
Are these now covered in any other tests? I couldn't find them in this PR but maybe they were already duplicated by an existing test?

Reply:
I manually added them for `with()`, but now I've added them for all methods.